src/main/resources/io/github/dgroup/arch4u/pmd/arch4u-template-ruleset.xml

```diff
@@ -77,4 +77,30 @@
       ]]>
     </example>
   </rule>
+
+  <rule name="DontCreateStringConstant"
```

Comment:
Its not a template rule, thus, please move it to `arch4u-ruleset.xml`.
Please also configure this rule for all static constants from MediaType class.